cypress/integration/problem_detailed_page.js

```diff
@@ -19,4 +19,29 @@ describe("Problem Detailed Page", () => {
     cy.get(`a[href="/problems/1"]`).first().click();
     cy.contains("Pobierz treść");
   });
+
+  const path = require("path");
+
+  it("Initiate download pdf file containing problem details and verify it", () => {
+    cy.get(`a[href="/problems/1"]`).first().click();
+
+    cy.get('[class*="Button-sc-1v3p2bo-0 kuukQO"]')
```

Comment:
You can use `contains` as well, would be more human friendly then:
```suggestion
    cy.contains('Pobierz treść').should("be.visible").click();
```